### PR TITLE
dmonitoringd: don't check for buttonEvents or vCruise change

### PR DESCRIPTION
**Pull request by ZwX1616**
carState is 100Hz and button is one frame so this check is very inconsistent

### CODE REVIEW

**Review by sshane — CHANGES_REQUESTED** [review]
‎

File: selfdrive/monitoring/dmonitoringd.py [hunk 1]
```diff
@@ -31,8 +31,7 @@ def dmonitoringd_thread():
     # Get interaction
     if sm.updated['carState']:
       v_cruise = sm['carState'].cruiseState.speed
```
Comment (sshane) [inline selfdrive/monitoring/dmonitoringd.py:33]: controlsState.vCruise is the superset of PCM and non-PCM cars. This currently doesn't capture Hyundai OP long for example

Comment (sshane) [inline selfdrive/monitoring/dmonitoringd.py:33]:
```suggestion
    if sm.updated['controlsState']:
      v_cruise = sm['controlsState'].vCruise
```

**Comment by adeebshihadeh** [comment]
@sshane how can we make the right fields to use more obvious here? Stuff like non-PCM shouldn't leak over into the DM daemon.

**Comment by ZwX1616** [comment]
> @sshane how can we make the right fields to use more obvious here? Stuff like non-PCM shouldn't leak over into the DM daemon.

I guess renaming the fields to like speedPcm helps

**Comment by sshane** [comment]
Can also rename `cruiseState` to be more explicit: `pcmState` (this info doesn't always come from the pcm though)

**Pull request merged**